package.json

```diff
@@ -20,7 +20,6 @@
     "docs:deploy": "storybook-to-ghpages",
     "docs:watch": "start-storybook -p 6006",
     "lint": "tslint --project .",
-    "postpublish": "npm run docs:build && npm run docs:deploy",
```

Comment:
reason? temporary for yalc or is this change permanent?

Reply:
yalc.